libs/ledgerjs/packages/types-live/src/transaction.ts

```diff
@@ -102,9 +102,19 @@ export type TransactionCommonRaw = {
 export type FeeStrategy = {
   amount: BigNumber;
   displayedAmount?: BigNumber;
+  txParameters?: FeeStrategyTxParameters;
   label: string;
   unit?: Unit;
 };
+
+/**
+ * Parameters are used to inform about specific transaction parameters
+ * Currently used for supplying EIP1559 fee information
+ */
+export type FeeStrategyTxParameters = {
+  maxBaseFeePerGas?: BigNumber;
+  maxPriorityFeePerGas?: BigNumber;
+};
```

Comment:
I don't think this belong in here because it's ethereum specific

Reply:
I see. I removed it, however wasn't sure where to place the type definitions for that, so for now it's just removed